sdk/core/ts-http-runtime/src/util/bytesEncoding.ts

```diff
@@ -1,6 +1,12 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
+declare global {
```

Comment:
shouldn't we not need this if our `@types/node` version is correct?

Reply:
Reverting change for now to move to a browser versus node implementation.